### PR TITLE
Updates from Kristine's science writer for simpler language

### PR DESCRIPTION
**Pull request by meredithslota**

### CODE REVIEW

File: articles/making_a_project.md [hunk 1]
```diff
@@ -7,28 +7,29 @@ order: 2
 ---
 
 
-# What is a Synapse Project? 
+# Projects in Synapse 
 
-Synapse `Projects` are online workspaces where researchers can collaborate and organize their work, regardless of whether they're working independently, collaborating in small working groups, or sharing their research across large consortia working together to solve big problems. Here are just a few things you can do with a Synapse Project: 
+`Projects` in Synapse serve as workspaces where researchers collaborate.  Researchers using Synapse can work in private or collaborate in groups. Some users share their research across large consortia. Projects support the following:
```
Comment (kdaily) [inline articles/making_a_project.md]:
```suggestion
`Projects` in Synapse serve as workspaces where researchers collaborate. Researchers using Synapse can work in private or collaborate in groups. Some users share their research across large consortia. Projects support the following:
```

File: articles/making_a_project.md [hunk 2]
```diff
@@ -7,28 +7,29 @@ order: 2
...
 
-* **Store Data, Code, &amp; Results**: You can upload your data, code, and results to Synapse, or store a reference to where they are on your local computer system. These digital assets can be stored as Files, hosted by Synapse or in your own external cloud storage, or using [Docker Containers](http://docs.synapse.org/articles/docker.html). 
+* **Store Data, Code, and Results**: You can upload your data, code, and results to Synapse, or store a reference to their location in your local computer system. Everything can be stored as Files, hosted by Synapse or in your own external cloud storage, or using [Docker Containers](http://docs.synapse.org/articles/docker.html).
```
Comment (kdaily) [inline articles/making_a_project.md]:
```suggestion
* **Store Data, Code, and Results**: You can upload your data, code, and results to Synapse, or store a reference to their location in your local computer system or on the web. Everything can be stored as Files, hosted by Synapse, in your own external cloud storage, or using [Docker Containers](http://docs.synapse.org/articles/docker.html). 
```

File: articles/making_a_project.md [hunk 3]
```diff
@@ -7,28 +7,29 @@ order: 2
...
 
-* **Store Data, Code, &amp; Results**: You can upload your data, code, and results to Synapse, or store a reference to where they are on your local computer system. These digital assets can be stored as Files, hosted by Synapse or in your own external cloud storage, or using [Docker Containers](http://docs.synapse.org/articles/docker.html). 
+* **Store Data, Code, and Results**: You can upload your data, code, and results to Synapse, or store a reference to their location in your local computer system. Everything can be stored as Files, hosted by Synapse or in your own external cloud storage, or using [Docker Containers](http://docs.synapse.org/articles/docker.html). 
 
-* **Control Access or Release Your Work Publicly**: You have complete fine-grained control over how users and groups can interact with your work. Work privately or collaborate openly, and everything in between.
+* **Control Access or Release Your Work Publicly**: You have complete control over how users and groups can interact with your work. Work privately or collaborate openly, and everything in between.
```
Comment (kdaily) [inline articles/making_a_project.md]:
```suggestion
* **Control Access or Release Your Work Publicly**: You have complete control over how users and groups can interact with your work. Work privately, openly, or somewhere in between.
```

File: articles/making_a_project.md [hunk 4]
```diff
@@ -7,28 +7,29 @@ order: 2
...
+* **Custom, Searchable Annotations**: Assign any key/value pair you want. Those values become searchable and available to those granted access.
 
-* **Full Programmatic API &amp; R Integration**: Synapse is built on a REST API, so you can interact with Synapse using any programming language. We have built out full featured clients for R and Java. 
+* **Full Programmatic API and Python/R Integration**: Synapse is built on a RESTful API, so you can interact with Synapse using any programming language. We have built out full-featured clients for Python, R, and Java. 
 
 * **Attach Figures and Documents**: Upload documents and images via the website or programmatically.
 
-* **Describe &amp; Version**: Describe each of the elements in your workflow, and then version those when you see fit. This allows you to explicitly record which version of the code you ran on which version of the data.
+* **Describe and Version**: Describe each of the elements in your workflow, and then practice version control when you see fit. This allows you to explicitly record which version of the code you ran on which version of the data.
```
Comment (kdaily) [inline articles/making_a_project.md]:
```suggestion
* **Describe and Version**: Describe each element in your workflow and practice version control when you see fit. This allows you to explicitly record which version of the code you ran on which version of the data.
```

File: articles/making_a_project.md [hunk 5]
```diff
@@ -7,28 +7,29 @@ order: 2
...
 
-* **Create Links &amp; See Usage**: See some data stored elsewhere on Synapse? Make a link to store it in your project. Want to see what projects are using your data? They are displayed automatically.
+* **Create Links and See Usage**: You can make a link to store data from elsewhere on Synapse
```
Comment (kdaily) [inline articles/making_a_project.md]:
```suggestion
```

Comment (kdaily) [reply articles/making_a_project.md]: added above in the Files that you can link

File: articles/making_a_project.md [hunk 6]
```diff
@@ -7,28 +7,29 @@ order: 2
...
+and use it in your project. Other projects using your data are displayed automatically.
```
Comment (kdaily) [inline articles/making_a_project.md]:
```suggestion
```

we don't do this?

**Review by kdaily — CHANGES_REQUESTED** [review]

**Comment by kelshmo** [comment]
@kdaily I can't merge despite approving your commits for some reason...

**Comment by kdaily** [comment]
@kelshmo try again, updated your permissions.

**Review by kelshmo — APPROVED** [review]

**Pull request merged**